sysml.library/Domain Libraries/Quantities and Units/QuantityCalculations.sysml

```diff
@@ -4,6 +4,13 @@ package QuantityCalculations {
     import UnitsAndScales::ScalarMeasurementReference;
     import UnitsAndScales::DimensionOneValue;
     
+    calc def isZero :> NumericalFunctions::isZero(x: ScalarQuantityValue): Boolean {
+    	NumericalFunctions::isZero(x.num)
+    }
+    calc def isUnit :> NumericalFunctions::isUnit(x: ScalarQuantityValue): Boolean {
+    	NumericalFunctions::isUnit(x.num)
+    }
+    
```

Comment:
W.r.t. naming convention we have a mix of calc defs / functions using PascalCase and camelCase. I guess in line with the rest of the definitions we should use PascalCase. Perhaps raise as a future maintenance issue.

Reply:
Yes, we have been very inconsistent about this, which becomes more noticeable as we add more function libraries. On the one hand, using initial-uppercase names is consistent with the usual convention for classifiers/definitions. On the other hand, it is inconsistent with usual practice for mathematical functions like `sin` and `cos` and for functions adapted from OCL, like `includes` and `select`.

I had been thinking that we need to resolve this all together in a future issue, as you suggest. However, there are now inconsistencies even within a number of packages (e.g., `NumericalFunctions` has `isZero`, `isUnit`, `sum` and `product`, but `Abs`, `Min` and `Max`). I don't think we can fix everything this time around, but maybe I can at least revised the packages touched by this PR to use a consistent naming convention. I would propose to use initial-_lowercase_, because the functions in question are then mostly mathematical.

Reply:
Yes, agreed

Reply:
Updated in commit [23247b9](https://github.com/Systems-Modeling/SysML-v2-Pilot-Implementation/pull/337/commits/23247b9ff208ee7b43e70f066ef41795c5ff5fd8).